cmd/stack_package.go

```diff
@@ -75,13 +75,15 @@ func newStackPackageCmd(rootConfig *RootCommandConfig) *cobra.Command {
 
 	var stackPackageCmd = &cobra.Command{
 		Use:   "package",
-		Short: "Package a stack in the local Appsody environment",
-		Long: `This command is a tool for stack developers to package a stack from their local Appsody development environment. Once the stack is packaged it can then be tested via Appsody commands. The package command performs the following:
-- Creates/updates an index file named "dev.local-index.yaml" and stores it in .appsody/stacks/dev.local
-- Creates a tar.gz for each stack template and stores it in .appsody/stacks/dev.local
-- Builds a Docker image named "dev.local/[stack name]:SNAPSHOT"
-- Creates an Appsody repository named "dev.local"
-- Adds/updates the "dev.local" repository of your Appsody configuration`,
+		Short: "Package your stack.",
```

Comment:
Very minor - do short descriptions usually have a full stop at the end? As the others in this change set don't.

Reply:
I think they all have here. Am I missing something?

Reply:
Currently some do and some don't, but we are adding a full stops to the short descriptions for all commands as part of this refactoring

Reply:
I think I'm going mad, ignore me. Agreed, let's put full stops on all.